modules/microshift-backing-up-manually.adoc

```diff
@@ -0,0 +1,50 @@
+//Module included in the following assemblies:
+//
+// * microshift_updating/microshift-update-options.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-backing-up-manually_{context}"]
+= Backing up {product-title} data manually
+
+You can back up {product-title} data manually at any time. Data backups are advised prior to any system updates. Backups are created in the `/var/lib/microshift-backups` directory by default. You can also specify a directory by adding custom parameters to the required backup command.
+
+[NOTE]
+====
+RPM-OSTree systems backup and restore data automatically.
+====
+
+.Prerequisites
+* You have root access to the host.
+* {product-title} and related workloads are stopped.
+
+.Procedure
+. Manually create a backup using the default name and parent directory, `/var/lib/microshift-backups/<default-backup-name>`, by running the following command:
++
+[source, terminal]
+----
+$ sudo microshift backup
+----
+
+. Optional. Manually create a backup with a specific name in the default directory by running the following command:
++
+[source, terminal]
+----
+$ sudo microshift backup --name /var/lib/microshift-backups/<my-custom-backup>/
+----
+
+. Optional. Manually create a backup in a specific parent directory by running the following command:
++
+[source, terminal]
+----
+$ sudo microshift backup --storage /var/lib/<custom-storage-location>
```

Comment:
Technically, it doesn't strictly needs to be in the `/var/lib/` (though the gain of Copy-on-Write could be lost)